pom.xml

```diff
@@ -29,8 +29,15 @@
       <id>pepov</id>
       <name>Peter Wilcsinszky</name>
     </developer>
+    <developer>
+      <name>Jens Mittag</name>
```

Comment:
Do you really plan to be developer for this project? If you are contributing single change and would handle and support it later, don't yourself as developer.